sdk/src/wallet/operations/syncing/transactions.rs

```diff
@@ -104,7 +104,9 @@ where
                         if let Some(tx_state) = metadata.transaction_state {
                             match tx_state {
                                 // TODO: Separate TransactionState::Finalized?
-                                TransactionState::Finalized | TransactionState::Confirmed => {
+                                TransactionState::Accepted
+                                | TransactionState::Confirmed
+                                | TransactionState::Finalized => {
                                     log::debug!(
                                         "[SYNC] confirmed transaction {transaction_id} in block {}",
                                         metadata.block_id
```

Comment:
Can you explain why this applies to "accepted" as well? Before writing this comment I tried find the answer myself in the go code, but I am still not entirely sure tbh.

Reply:
I don't actually know 🤷‍♀️ was hoping someone could tell me

Reply:
I wouldn't add it in the match here, because accepted could still get not finalized at the end
But probably good idea to put a TODO, since it might be interesting to match and do something in the accepted case already
From the indexer we also get outputs already when a tx gets accepted, but the outputs could also never get created at the end, so we definitely need to change some logic in the wallet around this